source/quick-start.txt

```diff
@@ -7,193 +7,182 @@ Quick Start
-<sample-data/load-sample-data/>` to run the examples in this guide.
+  npm install mongodb
 
-Store Configuration Separately
-------------------------------
+This command downloads ``mongodb`` package and dependencies required for its
+installation and saves them into a directory called ``node_modules`` in
+your project directory, and records the dependency information in the
+``package.json`` file you generated in the previous step.
 
-You can store your application configuration separately for security
-and convenience. We use the `dotenv
-<https://www.npmjs.com/package/dotenv>`_ NPM module to store our
-configuration data in a separate local file called ``.env``. If you are
-using git source control, add the ``.env`` file to your ``.gitignore``
-entries to prevent accidentally sharing contents such as database
-credentials.
+At this point, you should have appropriate versions of Node.js and NPM
+installed as well as a project directory that contains the dependencies you
+need to use the Node.js MongoDB driver.
 
-First, add ``dotenv`` to your project:
+Create a MongoDB Cluster
+------------------------
 
-.. code-block:: sh
+Set up a Free Tier Cluster in Atlas
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-  npm install --save dotenv
+After installing the Node MongoDB driver, create a MongoDB instance to store
+and manage your data. Go through the
+:atlas:`Get Started with Atlas <getting-started>` guide to set up a new
+Atlas account, free tier cluster (MongoDB instance), load datasets, and
+interact with the data.
 
-Next, create the ``.env`` file and add a variable called
-``MONGODB_URI``:
-
-.. code-block:: sh
-
-  touch .env
-  echo "MONGODB_URI=<connection_uri>" >> .env
-
-Add your Atlas Connection String
---------------------------------
-
-Open your newly created ``.env`` file. If you are unable to locate this
-file in your project directory, you may need to set your system to
-display hidden files. Once your ``.env`` is open, replace
-``<connection_uri>`` with your connection string from Atlas. Ensure you
-update ``<username>`` and ``<password>`` in your connection string.
+After completing the steps in the Atlas guide, you should have a new MongoDB
+cluster deployed in Atlas, a new database user, and sample datasets loaded
+into your cluster.
 
 Connect to your Cluster
 -----------------------
 
-For the final step, we'll write some code that tests your environment's
-connection to your instance of MongoDB.
-
-Create a file in your project directory called ``index.js``:
+In this step, we create and run an application that uses the Node.js MongoDB
+driver to connect to your instance of MongoDB and run a query on the sample
+data.
 
-.. code-block:: sh
+To connect to Atlas using the Node.js driver, we provide instructions to the
+driver on where and how to connect to your instance in a string called the
+*connection string*.
 
-  touch index.js
+Follow the
+:atlas:`Connect via Driver <driver-connection>` guide for instructions on
```

Comment:
I don't think this is the best page to link users to, as it doesn't have any screenshots and, unfortunately, it's not straightforward (from a UI perspective) to get the URI string. What about https://docs.atlas.mongodb.com/tutorial/connect-to-your-cluster/?

Reply:
You don't think they would be more confused by needing to install the Python driver or mongo shell on that page?

Reply:
I think they would be confused by both pages, but I think users will be confused less by pages that use screenshots. Personally, I'm averse to linking them to any page here as it doesn't seem like we have a good page to link them to. Can we add a screenshot here instead?

Reply:
Just to clarify, do you want screenshots of:
https://docs.atlas.mongodb.com/tutorial/connect-to-your-cluster/#id2
and
https://docs.atlas.mongodb.com/tutorial/connect-to-your-cluster/#copy-the-provided-connection-string-from-the-connection-string-only-dialog-tab
?

Reply:
Yeah, what do you think?

The first screenshot is helpful. The second screenshot is helpful too, but there is a modal that a user needs to go through to get from the first screenshot to the second.